checkerScripts.js

```diff
@@ -105,23 +105,28 @@ var selectedPiece = null;
 var rigthPiecePlace = null;
 var leftPiecePlace = null;
 var selectedPieceColor = null;
+var pieceRemove = null;
+var handPlay = 1;
 
 function getPieceMovement(){
 	
 	var coordX = parseInt($(this).attr('name')[0]);
 	var coordY = parseInt($(this).attr('name')[1]);
-	
-	if(board[coordX][coordY] == 1){
-        selectedPieceColor = 1;
-		getleftRightIndex(coordX, coordY);
-		
 
-	}else if(board[coordX][coordY] == -1){
-        selectedPieceColor = -1;
-		getleftRightIndex(coordX, coordY);
-		
-	}
+    if(handPlay == 1)
+    	if(board[coordX][coordY] == 1){
+
+            selectedPieceColor = 1;
+    		getleftRightIndex(coordX, coordY);
+    	}
+
+    if(handPlay == -1)  
+        if(board[coordX][coordY] == -1){
+            selectedPieceColor = -1;
+    		getleftRightIndex(coordX, coordY);
+    	}
```

Comment:
I think you don't need to ask for `hadPlay` variable, you can only assign the value to `selectedPieceColor` and call the function `getleftRightIndex`.  the `handPlay` will always be `1` or `-1` value.

**i.e.:**
```javascript
selectedPieceColor = handPlay;
getleftRightIndex(coordX, coordY);
```